source/agora/consensus/data/genesis/Coinnet.d

```diff
@@ -34,7 +34,7 @@ public immutable Block GenesisBlock =
         height:      Height(0),
         merkle_root: GenesisMerkleRoot,
         enrollments: Enrollments,
-        validators:  BitField!ubyte(6),
+        validators:  BitMask(0),
```

Comment:
Should probably still be 6

Reply:
But they are not `active` until block 1. They just have enrolments in Genesis block and do not sign.

Reply:
Good point! Ignore then. Or perhaps add your explanation as a comment ?